src/service/template-impl.js

```diff
@@ -165,14 +41,14 @@ export class Templates {
 
     /**
      * A map from template type to template's class promise.
-     * @private @const {!Object<string, !Promise<!TemplateClassDef>>}
```

Comment:
I don't think BaseTemplate exists here? Don't you need to reference the file path?

Reply:
Oh right, like `typeof ../base-template.BaseTepmlate` right?